src/main/java/com/google/sps/servlets/ReasonServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Arrays;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that sends reason for use to /text. */
+@WebServlet("/text")
+public final class ReasonServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "reason", "");
+    System.out.println(text);
+    String[] words = text.split("\\s*,\\s*");
+    response.setContentType("text/html;");
+    response.getWriter().println(Arrays.toString(words));
+  }
+
+  /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
+   */
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    String value = request.getParameter(name);
+    if (value == null) {
+      return defaultValue;
+    }
+    return value;
+  } 
+}
+
+    // boolean upperCase = Boolean.parseBoolean(getParameter(request, "upper-case", "false"));
```

Comment:
I like these ideas, but you could probably remove this from your code? :)

Reply:
I put them on a doc now, I should have thought about that sooner